features/steps/step_selenium_alias_hipchat.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: UTF-8 -*-
+
+# @mark.steps
+# ----------------------------------------------------------------------------
+# STEPS:
+# ----------------------------------------------------------------------------
+from behave import when, then
+
+
+@when('we create new alias in chat')
+def step_impl(context):
+
+    context.lobby_page.navigate()
+    context.lobby_page.random_click()
+    context.lobby_page.open_alias_room()
+    assert context.lobby_page.chat_adding_alias()
+
+@when('we create new alias with click buttons')
+def step_impl(context):
+    context.lobby_page.delete_alias_in_chat_comands()
+    context.lobby_page.open_alias_menu()
+    context.lobby_page.open_menu()
+    context.lobby_page.switch_to_frame()
```

Comment:
In ideal world, steps should not be aware we are in browser.
Page object is handling anything browser-related.
Test (step) is operating with abstractions that are implementation-agnostic.
How would you name this call if we were using desktop client?